manifests/config.pp

```diff
@@ -10,6 +10,25 @@
 
   $_ip = join($ferm::ip_versions, ' ')
 
+  if $ferm::install_method == 'vcsrepo' and $ferm::manage_service {
+    include systemd::systemctl::daemon_reload
+
+    file_line { "ferm.service use ${ferm::configfile} to start":
```

Comment:
instead of modyfiying the default unit file, we should always replace it with a version in /etc, or for this specific usecase, use a drop_in: https://github.com/camptocamp/puppet-systemd#drop-in-files

Reply:
can you confirm this is now addressed ?